### DISCUSSION TITLE
awc http client - ignore ssl

### DISCUSSION THREAD

**Discussion by jRimbault** [actix/actix-web#3529] [Q&A] [answered]
Hi, is there a way to instruct the [`awc::Client`](https://docs.rs/awc/3.5.1/awc/struct.Client.html) to ignore invalid certificates?

**Comment by robjtede** [comment, marked as answer]
yes, there's a code for this in one of our test cases:

https://github.com/actix/actix-web/blob/34327bd2217bdccdff030198e9d5aa0d6e3a8e82/awc/tests/test_rustls_client.rs#L53-L91
